connector/oidc/oidc.go

```diff
@@ -284,12 +285,29 @@ func (c *oidcConnector) Refresh(ctx context.Context, s connector.Scopes, identit
 	return c.createIdentity(ctx, identity, token, refreshCaller)
 }
 
+func (c *oidcConnector) TokenIdentity(ctx context.Context, subjectTokenType, subjectToken string) (connector.Identity, error) {
+	var identity connector.Identity
+	token := &oauth2.Token{
+		AccessToken: subjectToken,
+	}
+	if subjectTokenType == "urn:ietf:params:oauth:token-type:access_token" {
```

Comment:
They're currently consts in server/ , but server imports the connectors, so exporting them there would be an import cycle.
Is there a more appropriate place to export them?

Reply:
I added this so that createIdentity called below would attempt to verify the token and get claims.

I think I need to restructure this part a bit: given an exchange caller, we must always verify the token first.
I've made the exchange verifier block a separate condition in createIdentity